### DISCUSSION TITLE
Floating point y-axis

### DISCUSSION THREAD

**Discussion by kennysimsj** [h2oai/wave#1522] [Q&A] [answered]
Is there a way to have the y-axis of the plot to be rounded up to 2 decimal points?

![image](https://user-images.githubusercontent.com/82559807/176088547-10ee5d42-0f20-4f84-aabe-edf7b0757115.png)

**Comment by lo5** [comment, marked as answer]
Yes. Instead of `y='=shap'`, pass `y='=${{intl shap minimum_fraction_digits=2 maximum_fraction_digits=2}}'`.

Additional formatting options for the `intl()` function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat

(Pass snake_case instead of camelCase).
